liara_nginx.conf

```
@@ -65,6 +65,10 @@ location ~ ^/clients/api(.*)?$ {
   return 301 /client-api/about;
 }
 
+location ~ ^/managing-multiple-acounts(.*)?$ {
```

I guess this is enough:

```suggestion
location ~ ^/managing-multiple-acounts/?$ {
```